docs/topics/api/signing.rst

```diff
@@ -38,10 +42,10 @@ validation and you will be able to check its status.
 
     .. sourcecode:: bash
 
-        curl https://addons.mozilla.org/api/v3/addons/my-addon/versions/1.0/
-            -XPUT --form 'upload=@build/my-addon.xpi' -H 'Authorization: JWT <jwt-token>'
+        curl https://addons.mozilla.org/api/v3/addons/@my-addon/versions/1.0/
+            -g -XPUT --form 'upload=@build/my-addon.xpi'
+            -H 'Authorization: JWT <jwt-token>'
 
-    :param addon-guid: the GUID for the add-on.
```

Comment:
should it be @my-addon? and removing this line means that we don't define what @my-addon is?